internal/pkg/runtime/platform/vmware/vmware.go

```diff
@@ -83,7 +83,8 @@ func (v *VMware) ExternalIPs() (addrs []net.IP, err error) {
 // KernelArgs implements the runtime.Platform interface.
 func (v *VMware) KernelArgs() kernel.Parameters {
 	return []*kernel.Parameter{
-		kernel.NewParameter("console").Append("ttyS0"),
+		kernel.NewParameter("console").Append("tty0"),
 		kernel.NewParameter("earlyprintk").Append("ttyS0,115200"),
+		kernel.NewParameter(constants.KernelParamConfig).Append(constants.ConfigGuestInfo),
```

Comment:
I think we should set this in our official OVA using the Makefile. It is possible that a user could not want to use guest info for the config. VMware is sort of one of the special cases, like KVM, where there is more than one way you can get the config to the machine.

Reply:
Maybe the Makefile is a bad place for it. I like the idea of not setting this if the user supplies something different.

Reply:
since this is specific to the VMWare platform i don't think there's any harm here.

Reply:
The harm is that `guestinfo` is optional, and that VMware allows for more than one way to configure it.

Reply:
Also, VMware becomes the snowflake here. No other provider is supplying this arg, because it is meant to be supplied on the frontend via the installer.